sagemaker-geospatial/digital-farming-pipelines/digital-farming-sagemaker-geospatial-part-1.ipynb

```diff
@@ -31,9 +31,9 @@
    },
```

Comment:
<p>Don't hardcode regions as you don't know where the customer is running the notebook from - it is preferable to use <code>boto3.Session().region_name</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3699/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>